commands/ListRoutes/index.ts

```diff
@@ -175,5 +176,7 @@ export default class ListRoutes extends BaseCommand {
     } else {
       new RoutesPrettyRenderer(this).render()
     }
+    
+    await this.application.shutdown()
```

Comment:
Shouldn't we use await this.exit()? I didn't understand the difference, I'll have to dig into the code

Reply:
The `await this.exit()` will exit the process explicitly. Whereas, the `this.application.shutdown()` notifies all the providers to start graceful shutdown.

If the issue is that `process.exit` in Node.js interrupts the `stdout`, then we cannot use `await this.exit()`.

Reply:
Ok thanks that totally makes sense !
Indeed, by replacing with `this.exit`, the same problem happens